.kokoro/requirements.in

```diff
@@ -8,6 +8,3 @@ setuptools
 nox>=2022.11.21 # required to remove dependency on py
 charset-normalizer<3
 click<8.1.0
-sphinx==4.5.0 # docs PDF see noxfile for HTML docs build
```

Comment:
Maybe move these to `release-nightly.sh` or just remove the PDF build from `release-nightly.sh`